airflow/bin/cli.py

```diff
@@ -118,29 +118,32 @@ def backfill(args, dag=None):
 
 
 def trigger_dag(args):
-    session = settings.Session()
-    # TODO: verify dag_id
+    dag = get_dag(args)
+
+    if not dag:
+        logging.error("Cannot find dag {}".format(args.dag_id))
+        sys.exit(1)
```

Comment:
@aoen I agree, but the full change should be part of a different PR I think. I can change to raising and create a Jira for it?